.gitlab/packaging/rpm.yml

```diff
@@ -1,13 +1,13 @@
 ---
 
-.updater_package_rpm_common:
+.installer_package_rpm_common:
   rules:
     - !reference [.except_mergequeue]
     - when: on_success
   before_script:
     - source /root/.bashrc
     - export RELEASE_VERSION=$RELEASE_VERSION_7
-    - export INSTALL_DIR=/opt/datadog/updater/
+    - export INSTALL_DIR=/opt/datadog/installer/
```

Comment:
this needs to be deleted as it's moved to another dir here: https://github.com/DataDog/datadog-agent/pull/24461